app/src/main/res/values/strings.xml

```diff
@@ -171,91 +171,92 @@
     </plurals>
     <string name="navigate">Navigate</string>
     <string name="verified">verified</string>
-    <string name="verified_desc">Charger verified by a member at the %s community — not necessarily working right now.</string>
+    <string name="verified_desc">Charger verified by a member of the %s community. (Not necessarily working now.)</string>
     <string name="charge_price_format">%2$s%1$.2f</string>
     <string name="charge_price_average_format">⌀ %2$s%1$.2f/kWh</string>
     <string name="charge_price_kwh_format">%2$s%1$.2f/kWh</string>
     <string name="chargeprice_select_connector">Choose connector</string>
-    <string name="chargeprice_provider_customer_tariff">Only for provider customers</string>
-    <string name="edit_on_goingelectric_info">If only an empty page is showing here, please first log in to GoingElectric.de.</string>
+    <string name="chargeprice_provider_customer_tariff">Only for tie-in customers</string>
+    <string name="edit_on_goingelectric_info">Please log in at GoingElectric.de if this page is empty.</string>
     <string name="percent_format">%.0f%%</string>
     <string name="chargeprice_session_fee">session fee</string>
     <string name="chargeprice_per_kwh">per kWh</string>
     <string name="chargeprice_per_minute">per min</string>
     <string name="chargeprice_blocking_fee">Blocking fee >%s</string>
-    <string name="chargeprice_no_tariffs_found">Chargeprice.app found no charging plans compatible with this charger.</string>
+    <string name="chargeprice_no_tariffs_found">No charging plans for this charger on chargeprice.app.</string>
     <string name="powered_by_chargeprice">powered by Chargeprice</string>
-    <string name="chargeprice_base_fee">Base fee: %2$s%1$.2f/month</string>
-    <string name="chargeprice_min_spend">Minimum spend: %2$s%1$.2f/month</string>
+    <string name="chargeprice_base_fee">Base cost: %2$s%1$.2f/month</string>
```

Comment:
```suggestion
    <string name="chargeprice_base_fee">Subscription: %2$s%1$.2f/month</string>
```
I am thinking "gebühr/fee" is more poised towards incurred cost through some wrongdoing.